scripts/zones/Sea_Serpent_Grotto/npcs/Shady_Sconce.lua

```diff
@@ -0,0 +1,25 @@
+-----------------------------------
+-- Area: Sea Serpent Grotto
+--  NPC: Shady Sconce
+-- !pos
+-----------------------------------
+local ID = require("scripts/zones/Sea_Serpent_Grotto/IDs")
+-----------------------------------
+
+function onTrade(player, npc, trade)
+end
+
+function onTrigger(player, npc)
+    if player:getCurrentMission(AMK) == dsp.mission.id.amk.LENDER_BEWARE_READ_THE_FINE_PRINT and player:getVar("AMK8") == 0 then
```

Comment:
I'll extrapolate in main discussion, but as an example, if you tweak your var usage to "stage-based" (here: AMK8 == 1), you could do things like drop currentMission checks